app/src/main/java/com/example/series_collector/ui/adapters/SeriesFollowedAdapter.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.series_collector.ui.adapters
+
+import android.content.Intent
+import android.net.Uri
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.findNavController
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.series_collector.data.Series
+import com.example.series_collector.databinding.ListItemSeriesFollowedBinding
+import com.example.series_collector.ui.Inventory.InventoryItemCallback
+import com.example.series_collector.ui.home.HomeFragmentDirections
+
+class SeriesFollowedAdapter(
+    private val inventoryItemCallback: InventoryItemCallback
+) : ListAdapter<Series, RecyclerView.ViewHolder>(SeriesFollowedDiffCallback()) {
```

Comment:
```suggestion
) : ListAdapter<Series, SeriesFollowedViewHolder>(SeriesFollowedDiffCallback()) {
```
이런식으로 viewholder type을 직접 사용하는방식도 있습니다 :)

Reply:
넵 참고하겠습니다!